Makefile

```diff
@@ -41,120 +54,120 @@ OUT=out/$(EMULATOR)
 
 $(OUT)/sources.tape: $(ITSTAR) build/$(EMULATOR)/stamp $(OUT)/syshst/$(H3TEXT)
-	mkdir -p $(OUT)
-	rm -f src/*/*~
-	touch -d 1981-10-06T19:03:37 'bin/emacs/einit.:ej'
-	touch -d 1981-09-19T21:42:56 'bin/emacs/[pure].162'
-	touch -d 1981-03-31T20:41:45 'bin/emacs/[prfy].173'
+	$(MKDIR) -p $(OUT)
+	$(RM) -f src/*/*~
+	$(TOUCH) -d 1981-10-06T19:03:37 'bin/emacs/einit.:ej'
+	$(TOUCH) -d 1981-09-19T21:42:56 'bin/emacs/[pure].162'
+	$(TOUCH) -d 1981-03-31T20:41:45 'bin/emacs/[prfy].173'
 	cd src; $(ITSTAR) -cf ../$@ $(SRC)
 	cd doc; $(ITSTAR) -rf ../$@ $(DOC)
 	cd bin; $(ITSTAR) -rf ../$@ $(BIN)
 	cd $(OUT); $(ITSTAR) -rf ../../$@ system syshst
 	-cd user; $(ITSTAR) -rf ../$@ *
 
 $(OUT)/salv.tape: $(WRITETAPE) $(RAM) $(NSALV)
-	mkdir -p $(OUT)
+	$(MKDIR) -p $(OUT)
 	$(WRITETAPE) -n 2560 $@ $(RAM) $(NSALV)
 
 $(OUT)/dskdmp.tape: $(WRITETAPE) $(RAM) $(DSKDMP)
-	mkdir -p $(OUT)
+	$(MKDIR) -p $(OUT)
 	$(WRITETAPE) -n 2560 $@ $(RAM) $(DSKDMP)
 
 start: build/$(EMULATOR)/start
-	ln -s $< $*
+	$(LN) -s $< $*
 
 build/klh10/stamp: $(KLH10) start build/klh10/dskdmp.ini
-	mkdir -p $(OUT)/system
+	$(MKDIR) -p $(OUT)/system
 	cp=0; ca=0; \
-	test $(CHAOS) != no && cp=1 && ca=$(CHAOS); \
+	$(TEST) $(CHAOS) != no && cp=1 && ca=$(CHAOS); \
 	x=`echo $(IP) | tr . ,`; \
-	sed -e "s/%IP%/$$x/" \
+	$(SED) -e "s/%IP%/$$x/" \
 	    -e 's/%NETMASK%/$(NETMASK)/' \
 	    -e "s/%CHAOSP%/$$cp/" \
 	    -e "s/%CHAOSA%/$$ca/" < build/klh10/config.203 > $(OUT)/system/config.203
-	touch $@
+	$(TOUCH) $@
 
 build/simh/stamp: $(SIMH) start
-	mkdir -p $(OUT)/system
+	$(MKDIR) -p $(OUT)/system
 	cp build/simh/config.* $(OUT)/system
-	touch $@
+	$(TOUCH) $@
 
 build/sims/stamp: $(KA10) start
-	mkdir -p $(OUT)/system
+	$(MKDIR) -p $(OUT)/system
 	cp build/sims/config.* $(OUT)/system
-	touch $@
+	$(TOUCH) $@
 
 build/klh10/dskdmp.ini: build/klh10/dskdmp.txt Makefile
 	cp=';'; ca=''; \
-	test $(CHAOS) != no && cp='' && ca='myaddr=$(CHAOS) $(CHAFRIENDS)'; \
-	sed -e 's/%IP%/$(IP)/' \
+	$(TEST) $(CHAOS) != no && cp='' && ca='myaddr=$(CHAOS) $(CHAFRIENDS)'; \
+	$(SED) -e 's/%IP%/$(IP)/' \
 	    -e 's/%GW%/$(GW)/' \
 	    -e "s/%CHAOSP%/$$cp/" \
 	    -e "s|%CHAOSA%|$$ca|" < $< > $@
 
 $(OUT)/syshst/$(H3TEXT): build/$(H3TEXT)
-	mkdir -p $(OUT)/syshst
-	test $(CHAOS) != no && c="CHAOS $(CHAOS), "; \
-	sed -e 's/%IP%/$(IP)/' \
+	$(MKDIR) -p $(OUT)/syshst
+	$(TEST) $(CHAOS) != no && c="CHAOS $(CHAOS), "; \
+	$(SED) -e 's/%IP%/$(IP)/' \
 	    -e 's/%HOSTNAME%/$(HOSTNAME)/' \
 	    -e "s/%CHAOS%/$$c/" < $< > $@
-	cat conf/hosts >> $@
+	$(CAT) conf/hosts >> $@
 
 $(KLH10):
 	cd tools/klh10; \
 	./autogen.sh; \
-	mkdir tmp; \
+	$(MKDIR) tmp; \
 	cd tmp; \
 	export CONFFLAGS_USR=-DKLH10_DEV_DPTM03=0; \
-	../configure --bindir=${PWD}/build/klh10; \
-	make base-ks-its; \
-	make -C bld-ks-its install
+	../configure --bindir=$(PWD)/build/klh10; \
+	$(MAKE) base-ks-its; \
+	$(MAKE) -C bld-ks-its install
 
 $(SIMH):
-	cd tools/simh; make pdp10
+	cd tools/simh; $(MAKE) pdp10
```

Comment:
You could use `$(MAKE) -C tools/simh pdp10` here, as above.

Reply:
Fixed all applicable places.